### DISCUSSION TITLE
MS SQL Server as backend

### DISCUSSION THREAD

**Discussion by Stuv7CB** [grafana/grafana#51671] [General]
Hello

Is there any plans to support MS SQL Server as backend DB? Are there well known issues which prevent from supporting MS SQL?

**Comment by ying-jeanne** [comment]
> Hello
> 
> Is there any plans to support MS SQL Server as backend DB? Are there well known issues which prevent from supporting MS SQL?

Hey @Stuv7CB Do you have an usecase that you can't use any of sqlite/mysql/postgres in your environment and has to use the mssql instead?

The main problem of adding new db type into Grafana is that we need to implement/maintain every time the dialect of that database. So the discussion is long.

But if you have a real usecase that all other DBs are not accessible and in your PRD env only mssql could be available, that might worth the efforts.

Thanks,

**Comment by bergquist** [comment]
https://github.com/grafana/grafana/issues/8674#issuecomment-1410780088

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
